abstract-integers/src/nat_mod.rs

```diff
@@ -35,6 +42,11 @@ macro_rules! modular_integer {
                 $max
             }
 
+            pub fn declassify(self) -> BigInt {
```

Comment:
Hum why did you call this function `declassify` ?

Reply:
Well, so this relates to the question to why implementing `Numeric` etc. for `BigInt`.
If we want to write tests for `Numeric` and `Integer` in a generic way (as I started), we need to a public version math integers. But that doesn't exist. So I take `BigInt` as public version of any math integer and implement the traits for it as well. That allows testing all integers in the same way. This shouldn't be used in specs of course ...
I guess some `external` declaration are needed for that. But would you otherwise be ok with that?

Reply:
Thanks for the clarification ! Yes everything concerning `BigInt` should be marked as `external`, I'm fine with that